crates/core/tedge_api/src/workflow/mod.rs

```diff
@@ -103,9 +108,17 @@ pub enum OperationAction {
         OperationName,
         Option<ShellScript>,
         StateExcerpt,
-        BgExitHandlers,
+        ExecHandlers,
     ),
 
+    /// Trigger a built-in operation
+    ///
+    /// ```toml
+    /// operation = "<builtin:operation-name>"
+    /// on_exec = "<state>"
+    /// ```
+    BuiltInOperation(OperationName, ExecHandlers),
```

Comment:
As a first step, I introduced a specific `BuiltInOperation` case along the existing `Operation` case.

I will consider to the merge the two.

Reply:
We decided to go against this idea: a builtin operation can only be invoked from the same operation.